.github/ISSUE_TEMPLATE/backport_request.yml

```diff
@@ -0,0 +1,111 @@
+      placeholder: "hello"
+    validations:
+      required: true
+  - type: "input"
+    id: "unstable-version"
+    attributes:
+      label: "Version in Unstable"
+      description: "Please indicate the current version of the package in Nixpkgs Unstable."
+      placeholder: "1.0.1"
+    validations:
+      required: true
+  - type: "input"
+    id: "stable-version"
+    attributes:
+      label: "Version in Stable"
+      description: "Please indicate the current version of the package in Nixpkgs Stable."
+      placeholder: "1.0.0"
+    validations:
+      required: true
+  - type: "textarea"
+    id: "reasoning"
+    attributes:
+      label: "Reasoning for Backport"
+      description: "Please briefly explain why this backport fits the [release suitability guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#changes-acceptable-for-releases) and why you think this update should be backported."
+      placeholder: "This update should be backported because it only includes bugfixes, and no breaking changes."
+    validations:
+      required: false
+  - type: "textarea"
+    id: "maintainers"
+    attributes:
+      label: "Notify Maintainers"
+      description: |
+        Please mention the people who are in the `meta.maintainers` list of the offending package. This is done by prefixing the person's username with an '@' character. You can quickly go to the `meta` attribute set of a package by searching for it on the [NixOS Package Search](https://search.nixos.org/packages) and clicking the "📦 Source" button.
+
+        Please note that the maintainer attribute name does not always match the maintainer's GitHub username. If that occurs, try looking in [`maintainers/maintainer-list.nix`](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix) for the maintainer attribute name, and checking if the maintainer has a listed GitHub username.
+
+        If in doubt, check `git blame` for whoever last touched the package. Please add the mentions above the `---` characters.
+      value: |
+
+        ---
+
+        **Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)
+    validations:
+      required: false
+  - type: "checkboxes"
+    id: "sanity-check"
+    attributes:
+      label: "I assert that this issue is relevant for Nixpkgs"
+      description: |
+        This bug tracker is for actionable issues that have not [already been resolved](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+in%3Atitle+backport). Please ensure that this bug report is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+in%3Atitle+backport), and that you have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it while communicating in any NixOS space.
+      options:
+        - label: "I assert that this backport does not yet exist in an open pull request."
```

Comment:
Can we move the URLs onto the labels? Doing that would avoid the need of the big paragraph before the actual list itself, which I think many people would just skip because frankly it's not very prominent (compare the size of the paragraph to the list, though if this is a Refined GitHub thing then feel free to ignore me)

<img src="https://github.com/user-attachments/assets/a0e9a0f9-8268-4775-a0b8-4d9305f18b15" alt="Screenshot showing the size difference between the paragraph and the list" width=500>

Reply:
I'll check again. Markdown is prohibited on drop-down labels, so I assumed the same was true for checkboxes.

Reply:
I was incorrect. Markdown is allowed on checkboxes. I'll edit them now.